app/models/ability.rb

```diff
@@ -0,0 +1,20 @@
+class Ability
```

Comment:
- [ ] Based on the requirements of this project, a user can delete a post if it is theirs or if they have an `admin role`. To test this, I created two users A and B. Later, I update the role column of B to admin so that B can delete all the posts created by A but this was not the case. Please let's kindly fix it. Kindly check the comment above for this specific and tests if everything works as expected :+1: